news.xml

```diff
@@ -556,10 +556,10 @@ $ nix-store -qR /run/current-system | grep openssl
     </title>
     <description>
       <a href="https://www.domenkozar.com/">Domen Kožar</a> gave <a
-        href="https://fosdem.org/2014/schedule/event/nixos_declarative_configuration_linux_distribution/">a
```

Comment:
use official archive https://archive.fosdem.org/2014/schedule/event/nixos_declarative_configuration_linux_distribution/

Reply:
static site that won’t change